### ISSUE TITLE
Webmail: 503 Service Unavailable

### ISSUE THREAD

**Issue by johannes73** [msimerson/Mail-Toaster-6#336]
I am probably doing a stupid mistake, but I have tried multiple times to find it, and am stuck.  

### Expected behavior

After upgrade from FreeBSD 10.3 to FreeBSD 11.1. In webmail interface, choose "Webmail" --> "Rainloop" or "Squirrelmail", and login screen should show. 

### Observed behavior

"Webmail" --> "Rainloop" and "Squirrelmail" shows:

"503 Service Unavailable
No server is available to handle this request."

jls
   JID  IP Address      Hostname                      Path
    25  172.16.15.3     dns                           /jails/dns
    26  172.16.15.4     mysql                         /jails/mysql
    27  172.16.15.8     vpopmail                      /jails/vpopmail
    28  172.16.15.15    dovecot                       /jails/dovecot
    29  172.16.15.10    webmail                       /jails/webmail
    30  172.16.15.12    haproxy                       /jails/haproxy
    31  172.16.15.5     clamav                        /jails/clamav
    32  172.16.15.16    redis                         /jails/redis
    33  172.16.15.13    rspamd                        /jails/rspamd
    34  172.16.15.17    geoip                         /jails/geoip
    35  172.16.15.9     haraka                        /jails/haraka
    36  172.16.15.11    monitor                       /jails/monitor
    37  172.16.15.35    rainloop                      /jails/rainloop
    38  172.16.15.34    squirrelmail                  /jails/squirrelmail
    39  172.16.15.48    dcc                           /jails/dcc

### Steps to reproduce

I upgraded server from FreeBSD 10.3 to FreeBSD 11.1, and provisioned all jails. 
Unprovisioned webmail, rainloop and squirrelmail. 
Deleted their entries from /etc/jails.conf.
Provisioned webmail, rainloop and squirrelmail again. 

Very thankful for any hints.

**Comment by msimerson** [comment]
Quick first guess: there's no config sections for roundcube in your haproxy config (legacy/upgrade issue). Solution: look at haproxy.conf in the provision-haproxy.sh script and update your config. Or if you have no customizations, nuke your /data/haproxy filesystem and reprovision haproxy.

**Comment by johannes73** [comment]
Thanks, Matt. You give super support. I deleted /data/haproxy, reprovisioned haproxy, and rebooted the server. But, strangely, the problem persisted. 

Then suddenly I could not SSH to the server. Rare. That made me suspicious, and after digging I found and commented out this line in /etc/rc.conf:

`# sshguard_enable="YES"`

After reboot, both webmail and SSHd have worked without problems. I am not sure what fixed the problem. Thanks again.
